web-app/src/pages/Dashboard/styled.js

```diff
@@ -2,11 +2,31 @@ import styled from 'styled-components';
 
 export const ButtonToggle = styled.div`
   position: absolute;
-  left: calc(60% - 111px);
+  right: 350px;
   top: 30px;
-  display: flex;
   height: 40px;
-   `;
+  display: none;
+  .btnText {
+    display: none;
+  }
+
+  span {
```

Comment:
Small thing but try to give everything a class rather than put CSS on a element.

Reply:
I don't think this is an issue when used with styled-components, maybe even preferable to keep code clean.

Reply:
It's not a stylistic reason, it's just for performance. A small thing but makes all the difference across an app. Appreciate it's CSS within JS but it should still stand. I've been too quite a few performance talks now and it's always brought up with CSS.

Reply:
Go with whatever is consistent to what already has been done but it's just good to note for future.

Reply:
@bgribz was beside me and mentioned he thinks a class is generated. I checked https://www.styled-components.com/docs/advanced and inspected a box and it does generate a class. So ignore what I said there in terms of styled components but the principle stands